src/plugins/condition/utils/evaluator.js

```diff
@@ -14,3 +14,19 @@ export const computeCondition = (resultMap, allMustBeTrue) => {
     }
     return result;
 };
+
+//Returns true only if limit number of results are satisfied
+export const computeConditionByLimit = (resultMap, limit) => {
+    let trueCount = 0;
+    for (let key in resultMap) {
+        if (resultMap.hasOwnProperty(key)) {
+            if (resultMap[key]) {
```

Comment:
I think that generally speaking just checking for "truthiness" is a bit vague and can lead to subtle bugs. I'd prefer to be explicit here and check `if (resultMap[key]) === true` or `if (resultMap[key]) !== undefined` - whatever the intended behavior is here.

Reply:
Done.